client/src/components/DateTime.test.ts

```diff
@@ -2,20 +2,25 @@ import { expect, test } from "vitest";
 import { mount } from "@vue/test-utils";
 import DateTime from "./DateTime.vue";
 
+const date = new Date();
+date.setHours(date.getHours() - 5);
+
 const props: any = {
-  dateTime: new Date().toISOString(),
+  dateTime: date.toISOString(),
   timeZone: "CET",
 };
 const options = { props };
 
 test("displays relative time", async () => {
   const wrapper = mount(DateTime, options);
-  expect(wrapper.text()).toContain("a few seconds ago");
+  expect(wrapper.text()).toContain("hours ago");
```

Comment:
Same here, make it work both with our local time and with the CI server time, which is apparently located/configured at a different timezone.